### PR TITLE
feat(Tests): Make DataHub Tests Feature configurable via env variable

### PR DESCRIPTION
**Pull request by jjoyce0510**
**Summary**
Configuration to make DataHub Tests Feature.

**Status**
Ready for review. 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-service/factories/src/main/resources/application.yml [hunk 1]
```diff
@@ -179,3 +179,6 @@ spring:
 springdoc:
   cache:
     disabled: true
+
+tests:
```
Comment (dexter-mh-lee) [inline metadata-service/factories/src/main/resources/application.yml:183]: is this clear enough? should we say datahubTests?

Comment (jjoyce0510) [reply metadata-service/factories/src/main/resources/application.yml:183]: metadataTests?

Comment (dexter-mh-lee) [reply metadata-service/factories/src/main/resources/application.yml:183]: Yeah. just anything more than simple tests which can also refer to unit tests

**Review by dexter-mh-lee — APPROVED** [review]
Minor comment. LGTM overall

**Pull request merged**